### PR TITLE
Update readme with Resource ACL

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -226,18 +226,29 @@ staging and dev environment.
 ### Resource
 
 For resource set the `partition` to `hca`, set your `service` name to the name of your component, and set the 
-`account-id` to the deployment stage. All other fields can be used as needed or use \* for wild cards. Resource names
- are case sensitive.
-
+`account-id` to the deployment stage. The field **resourcetype** can optionally be the name of a resource type defined
+ in fusillade. If `resourcetype` matches a resource type defined in fusillade then a resource policy will be used 
+ evaluating the users access. All other fields can be used as needed or use \* for wild cards. Resource names are
```
Comment (xbrianh) [inline Readme.md]: s/will be used evaluating the user access/will be used to evaluate user access/

File: Readme.md [hunk 2]
```diff
@@ -226,18 +226,29 @@ staging and dev environment.
...
-- arn:**hca**:**azul**:region:**dev**:resourcetype:resource
-- arn:**hca**:**matrix**:region:**staging**:resourcetype:resource:qualifier
-
+- arn:**hca**:**fusillade**:region:**dev**:**resourcetype**
+- arn:**hca**:**dss**:region:**staging**:**resourcetype**/resource
+- arn:**hca**:**query**:region:**integration**:**resourcetype**/resource/qualifier
+
+#### Resource ACL 
+A new resource type is created by providing the name of the resource type, and the actions that can be performed on it. 
+ Once a resource type is created you can store specific ids of the resource you'd like to control. A principal only has
```
Comment (xbrianh) [inline Readme.md]: > created you can store specific ids of the resource you'd like to control

To be consistent, s/ids/IDs/

File: Readme.md [hunk 3]
```diff
@@ -226,18 +226,29 @@ staging and dev environment.
...
+- arn:**hca**:**fusillade**:region:**dev**:**resourcetype**
+- arn:**hca**:**dss**:region:**staging**:**resourcetype**/resource
+- arn:**hca**:**query**:region:**integration**:**resourcetype**/resource/qualifier
+
+#### Resource ACL 
+A new resource type is created by providing the name of the resource type, and the actions that can be performed on it. 
+ Once a resource type is created you can store specific ids of the resource you'd like to control. A principal only has 
+ access to resource they are give access to, either directly or through group membership. The creator of a resource ID is
+ automatically designated as the owner of the resource. The owner of a resource can add additional owners, and assign 
+ access levels to their resource. The different levels of access are defined by access policies associated with a
```
Comment (xbrianh) [inline Readme.md]: > The different levels of access are defined by access policies associated

s/The different/Different

File: Readme.md [hunk 4]
```diff
@@ -226,18 +226,29 @@ staging and dev environment.
...
+
+#### Resource ACL 
+A new resource type is created by providing the name of the resource type, and the actions that can be performed on it. 
+ Once a resource type is created you can store specific ids of the resource you'd like to control. A principal only has 
+ access to resource they are give access to, either directly or through group membership. The creator of a resource ID is
+ automatically designated as the owner of the resource. The owner of a resource can add additional owners, and assign 
+ access levels to their resource. The different levels of access are defined by access policies associated with a 
+ resource type. New access policies can be defined for a resource type after the resource type has been created. All 
+ resources of that type share the same access policies and can only be assigned access policies that have already been 
+ defined. Access policies can only define policies that use actions supported by that resource type. Actions can be
```
Comment (xbrianh) [inline Readme.md]: > All resources of that type share the same access policies and can only be assigned access policies that have already been defined.

Trying to understand the implications of this sentence. Suppose I have two resources of type "foo," A and B. Is it impossible to assign different policies to A and B?

Alternatively, is there a pool of resource policies associated with "foo", and the policies attached to A and B must come from that pool?

File: Readme.md (hunk 1 above)
Comment (Bento007) [reply Readme.md]: will fix

File: Readme.md (hunk 2 above)
Comment (Bento007) [reply Readme.md]: will change to resource_id

File: Readme.md (hunk 4 above)
Comment (Bento007) [reply Readme.md]: The later. I'll change the description

**Pull request merged**